taxi_service/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for taxi_service project.
+
+Generated by 'django-admin startproject' using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+from django.conf.global_settings import AUTH_USER_MODEL
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-ckt==yq&*3h5l!71^ak9%@u!f=h$ieb^e*b)1b$y%to#gihs@f"

```

Comment:
The `SECRET_KEY` is hardcoded and should be kept secret in production. Consider using environment variables to manage sensitive information.